enterprise/server/remote_execution/runner/runner.go

```diff
@@ -1333,6 +1333,12 @@ func (p *pool) TryRecycle(ctx context.Context, r interfaces.Runner, finishedClea
 	if !cr.PlatformProperties.RecycleRunner {
 		return
 	}
+
+	exitEarly := true
+	if exitEarly {
+		return
+	}
```

Comment:
IIUC if you're never recycling, then the `bazel clean --expunge` will have no effect?

Reply:
Oh yeah that's a good point. I guess there are a couple different experiments to run:

1. Do not care if runner is recycled or not. Run `clean --expunge` every time. Dump kernel logs and see if there's anything interesting
2. Ensure we can trigger the segfault if we're always running on clean runners. Then try to trigger it on a recycled runner. See if that's possible or not

Reply:
I'll start with #1 and if that doesn't provide any helpful info go on to #2